src/main/java/org/duraspace/bagit/serialize/TarBagSerializer.java

```diff
@@ -35,15 +33,13 @@ public Path serialize(final Path root) throws IOException {
             final TarArchiveOutputStream tar = new TarArchiveOutputStream(os)) {
             tar.setLongFileMode(TarArchiveOutputStream.LONGFILE_POSIX);
             tar.setBigNumberMode(TarArchiveOutputStream.BIGNUMBER_POSIX);
-            final List<Path> files = Files.walk(root).collect(Collectors.toList());
+            final List<Path> files = Files.walk(root).toList();
```

Comment:
I'm not sure if this would end up biting us but `Files.walk` returns a Stream so we should put it in the try w/ resources above.

Reply:
Thanks for the feedback. I tried moving it to the try {} clause above, but there was a problem with it not being AutoCloseable. I'll look again to see if I missed something.